src/scripts/main.js

```diff
@@ -1,5 +1,32 @@
 'use strict';
 
 document.addEventListener('click', e => {
-  // write code here
+  const spider = document.querySelector('.spider');
+  const wall = document.querySelector('.wall');
+
+  if (e.target !== wall) {
+    return;
+  }
+
+  const x = e.offsetX;
+  const y = e.offsetY;
+
+  spider.style.left = x - spider.width / 2 + 'px';
```

Comment:
I've looked up your code and disagree with such approach. I'll explain. You have next stages:
1) Calculate coordinates (numbers)
2) Apply coordinates (convert to strings and apply styles, even they are incorrect)
3) Get coordinates (strings)
4) Convert 'em to numbers and check are in range
5) Apply new values

But you can do it much efficient by calculating and checking coordinates before applying to spider.